.travis.yml

```diff
@@ -3,9 +3,9 @@ sudo: false
 language: java
 
 jdk:
-  - openjdk6
```

Comment:
While jdk version < = 1.6, there is a Error:
com.google.gson.JsonSyntaxException: java.lang.NumberFormatException: For input string: "+1"
Did it mean "+1" cannot convert to Integer?